frontend/README.md

```diff
@@ -20,7 +20,7 @@ Change flutter to the version used in production (note that the below directory
 
 ```bash
 $ cd ~/snap/flutter/common/flutter
-$ git checkout 3.22.3
+$ git checkout 3.29.3
```

Comment:
Setting the new flutter version gave me some difficulty for some reason. Would be sweet if developing flutter in a docker container worked somehow.